frequencyChar.js

```diff
@@ -0,0 +1,20 @@
+function itemFrequency(array) {
+    let characterFrequency = {};
+    let totalCount = 0
+  
+    for (let item of array) {
+        totalCount+= 1
+      if (!characterFrequency[item]) {
+        // if not in the dictionary
+        characterFrequency[item] = 1;
+      } else {
+        characterFrequency[item] += 1;
+      }
+    } for (let keys of Object.keys(characterFrequency)) {
```

Comment:
Make sure you have ESLint set up and are listening to its feedback. This code is very hard for me to read.